nautobot/extras/templates/extras/customfield.html

```diff
@@ -0,0 +1,146 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load plugins %}
+{% load static %}
+{% load tz %}
+
+{% block header %}
```

Comment:
This will of course intersect with the generic view templates being introduced in #737 - just a FYI.